bootstrap/kernel_bootstrap.php

```diff
@@ -25,9 +25,18 @@
 }
 
 $xml = new \SimpleXMLElement(file_get_contents($phpUnitFile));
-$kernelDir = $xml->php[0]->server[0]['value'];
 
-$kernelFile = $rootDir.'/'.$kernelDir.'/AppKernel.php';
+$envDir = $xml->xpath("//php/server[@name='KERNEL_DIR']");
+if (!count($envDir)) {
+    throw new \Exception(
+        'Kernel path must be set via <server name"KERNEL_DIR" value="..."/>'
```

Comment:
@covex-nn we should have set `KERNEL_CLASS` here, as `KERNEL_DIR` is deprecated.

Reply:
ah, yes you do it some lines later :-)

Reply:
I think, that `KERNEL_DIR` is deprecated for usual apps, where it could be determined from `KERNEL_CLASS`. But here for now both values should be used, or only `KERNEL_DIR`

After all apps will be updated to use `KERNEL_CLASS` from phpunit.xml.dist, `KERNEL_DIR` will not be required and should be removed from `bootstrap/kernel_bootstrap.php`